datastream-kafka-connector/src/main/java/com/linkedin/datastream/connectors/kafka/AbstractKafkaConnector.java

```diff
@@ -101,6 +101,7 @@ public Thread newThread(@NotNull Runnable r) {
   enum DiagnosticsRequestType {
     DATASTREAM_STATE,
     PARTITIONS,
+    CONSUMER_OFFSETS
```

Comment:
This may be more of a question for @vishwajith-s and @ahmedahamid. What's the recommendation for reusing an existing `DiagnosticsRequestType` vs. adding a new one? I see that some request types return more than one piece of information (such as returning all partitions, paused partitions, auto-paused partitions, and inflight messages as part of a single request).